client/components/PubEdgeListing/PubEdgeListingCard.js

```diff
@@ -61,6 +61,10 @@ const PubEdgeListingCard = (props) => {
 	} = props;
 	const { communityData } = usePageContext();
 	const viewingFromTarget = getIsViewingFromTarget(pubEdge, viewingFromSibling, isInboundEdge);
+	const [hover, setHover] = useState(false);
+	const handleMouseEnter = useCallback(() => setHover(true), []);
+	const handleMouseLeave = useCallback(() => setHover(false), []);
+	const style = hover ? { borderColor: accentColor || communityData.accentColorDark } : {};
```

Comment:
I couldn't get my silly idea of `!important` working since there's no way to switch off a rule in a different scope in CSS, just override it with a new rule.